model_lab_configs/scripts/sanitize.py

```diff
@@ -14,6 +15,26 @@
 from model_lab import RuntimeEnum, RuntimeFeatureEnum
 from contextlib import contextmanager
 import re
+import inspect
+
+def printProcess(msg:str):
+    if(GlobalVars.verbose): print(f"Process {msg}")
+def printInfo(msg: str):
+    if(GlobalVars.verbose): print(msg)
+def printErrorAndExit(msg: str):
+    frame = inspect.currentframe().f_back
+    filename = os.path.relpath(frame.f_code.co_filename)
+    lineno = frame.f_lineno
+    # Red text, with file and line number, clickable in terminal
+    print(f"\033[31mERROR: {filename}:{lineno}: {msg}\033[0m")  
+    sys.exit(1)
```

Comment:
sometimes we don't want to fix one by one, so we should not fail once there is error

Reply:
it will be more like linter not a step-by-step debugger